mars/services/task/execution/ray/executor.py

```diff
@@ -258,6 +276,7 @@ async def execute_subtask_graph(
             elif output_count == 1:
                 output_object_refs = [output_object_refs]
             self._cur_stage_output_object_refs.extend(output_object_refs)
+            self._output_object_refs_nums.append(len(output_object_refs))
```

Comment:
The `output_object_refs` can be `None` if the `num_returns == 0`. Then, `len(None)` will raise `TypeError: object of type 'NoneType' has no len()`.

Reply:
If `num_returns == 0`, `output_count` is 0 and in this case it wont append the length.
But there is another problem if `output_count` is 0 that we couldn't cancel the ray task.

Reply:
This limiation can be commented. Also, the PR description can be updated, too.

Reply:
Is there any case whose `output_count` will be `0`?  I don't find anyone.

Reply:
Em, i think there is at least one output of a subtask.